testing_app/test/favorites_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+import 'package:testing_app/models/favorites.dart';
+import 'package:testing_app/screens/favorites.dart';
+
+Favorites favoritesList;
+
+Widget createFavoritesScreen() => ChangeNotifierProvider<Favorites>(
+      create: (context) {
+        favoritesList = Favorites();
+        return favoritesList;
+      },
+      child: MaterialApp(
+        home: FavoritesPage(),
+      ),
+    );
+
+void addRandomItems() {
+  for (var i = 0; i < 10; i += 2) {
```

Comment:
Any particular reason this is incrementing by two?

Reply:
I think that's something I did when I wasn't using random numbers and then forgot to change it later. =)